tests/run_test.sh

```diff
@@ -86,9 +86,17 @@ cp ${PATHRT}/../NEMS/src/conf/module-setup.sh.inc  module-setup.sh
 SRCD="${PATHTR}"
 RUND="${RUNDIR}"
 
-atparse < ${PATHRT}/fv3_conf/${FV3_RUN:-fv3_run.IN} > fv3_run
-
-atparse < ${PATHRT}/parm/${INPUT_NML:-input.nml.IN} > input.nml
+# FV3_RUN could have multiple entry seperated by space
```

Comment:
Could you explain how this multiple fv3_run item works? I see you have 2 fv3_run items in several of the tests.

Reply:
I basically loops through the given list and execute those scripts (under `fv3_conf` directory). The following as an example `export FV3_RUN="script1.IN script2.IN" that could be added to the specific test under `tests/`. The `script1.IN script2.IN` needs to match with the file names under `fv3_conf`.